server/src/main/java/org/apache/iotdb/db/metadata/mtree/MTreeBelowSG.java

```diff
@@ -536,7 +506,7 @@ public void createAlignedTimeseries(
   public Pair<PartialPath, IMeasurementMNode> deleteTimeseriesAndReturnEmptyStorageGroup(
```

Comment:
Does this method need concurrent control ?

Reply:
Fixed. I've synchronized all deletion methods. The ```deleteStorageGroup``` is synchronized in ```StorageGroupSchemaManager``` and ```deleteTimeseries``` is synchronized in belonged SGMManager.